tokio-epoll-uring/src/metrics.rs

```diff
@@ -1,29 +1,69 @@
-use std::sync::atomic::{AtomicU64, Ordering};
+use std::{
+    collections::HashMap,
+    sync::{
+        atomic::{AtomicU64, Ordering},
+        RwLock,
+    },
+};
+
+use once_cell::sync::Lazy;
 
 pub struct Metrics {
```

Comment:
How are going to access the metrics from the pageserver? The `SystemHandle` on the pageserver side doesn't have a reference to the metrics. We need something like that in order to report to Prometheus.

Reply:
Ah you can get them through `global()` I guess.

Reply:
Yea, decided to go with the current approach for now and export to pageserver from `global`.